src/trace/listener.ts

```diff
@@ -47,6 +50,7 @@ export class TraceListener {
     if (this.context) {
       options.tags = {
         cold_start: this.coldstart,
+        event_source: this.eventSource,
```

Comment:
What would you think about calling this something like `event_service` to leave open the possibility of making a tag like `event_resource` in the future that would be the ARN of the resource that triggered the function?

Reply:
Yeah that makes sense, or maybe "trigger_type", since it seems related to the concept of a trigger in the AWS Lambda UI, but triggers don't exist as a concept in the underlying infrastructure/resources.

Reply:
yeah "trigger_type" would make sense to me. @pinkerton want to weigh in?